captainslog.go

```diff
@@ -17,6 +17,18 @@ const (
 	CmdStop ChannelerCmd = iota
 )
 
+// MatchType represents types of matches that can
+// be made against a SyslogMsg
+type MatchType int
+
+const (
+	// Program match is an exact match against syslog program name
+	Program MatchType = iota
+
+	// Contains match checks if the syslog content contains a string
+	Contains
```

Comment:
There's not much protecting the meaning of `Contains` being related to `Program` here. I'd probably put them on adjacent lines, with a small comment aligned to the right of each item for clarity. For enum-like types, the name should be self-explanatory (since it will stand alone in the code as well). Usually if a comment is needed to define the enum, it'll be confusing in the actual code too.